--- exrex.py	2023-01-28 17:28:33.000000000 +0000
+++ exrex_new.py	2023-01-28 17:32:50.809769908 +0000
@@ -22,7 +22,11 @@
     from future_builtins import map, range
 except:
     pass
-from re import sre_parse, U
+try:
+    import re._parser as sre_parse
+except ImportError: # Python < 3.11
+    from re import sre_parse
+from re import U
 from itertools import tee
 from random import choice, randint
 from types import GeneratorType
